app/controllers/hyrax/leases_controller.rb

```diff
@@ -1,5 +1,55 @@
 module Hyrax
   class LeasesController < ApplicationController
-    include Hyrax::LeasesControllerBehavior
+    include Hyrax::Collections::AcceptsBatches
+
+    def index
+      authorize! :index, Hydra::AccessControls::Lease
+    end
+
+    def edit
+      work = find_resource(params[:id])
+      authorize! :edit, work
+      @resource = find_resource(work.lease_id)
+      @change_set = LeaseChangeSet.new(@resource, parent_resource: work)
+    end
+
+    # Removes a single lease
+    def destroy
+      curation_concern = find_resource(params[:id])
+      authorize! :destroy, curation_concern
+      lease = Hyrax::Actors::LeaseActor.new(curation_concern).destroy
+      flash[:notice] = lease.lease_history.last
+      if curation_concern.work? && curation_concern.file_sets.present?
+        redirect_to confirm_permission_path
+      else
+        redirect_to edit_lease_path
```

Comment:
Should this include an `:id`?

Reply:
This code has just been moved from a module to a class, I haven't change it.